newIDE/app/public/external/piskel/piskel-main.js

```diff
@@ -270,7 +142,7 @@ function piskelCreateAnimation(pskl, piskelOptions) {
     piskel.description,
     true
   );
-  pskl.utils.serialization.Deserializer.deserialize(sprite, function(piskel) {
+  pskl.utils.serialization.Deserializer.deserialize(sprite, function (piskel) {
```

Comment:
nitpicking: You can use an arrow function here.

Reply:
oops, I forgot one